samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/assessment/SecureDeliverySeb.java

```diff
@@ -351,6 +357,9 @@ private boolean publishConfigUpload(PublishedAssessmentIfc assessment, HttpServl
                 fileHandler.save(outputStream);
                 outputStream.flush();
 
+                // Push advisor, beacuse instructors can't add public resources normally
+                securityService.pushAdvisor(alwaysAllowSecurityAdvisor);
```

Comment:
I feel this should be done in a try/catch finally block, am I right?

Did you try this as instructor? Now works fine?

Reply:
It's all in a try/catch already, reveal a few lines above. Yes it's working fine for me now.

Reply:
niceeeeee